Example/BonMot/Cells/TagStylesCell.m

```diff
@@ -0,0 +1,40 @@
+//
+//  TagStylesCell.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/3/16.
+//  Copyright © 2016 Zev Eisenberg. All rights reserved.
+//
+
+#import "TagStylesCell.h"
+
+@interface TagStylesCell ()
+
+@property (weak, nonatomic) IBOutlet UILabel *label;
+
+@end
+
+@implementation TagStylesCell
+
++ (NSString *)title
+{
+    return @"Tag Styles";
+}
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+
+    BONChain *boldChain = BONChain.new.fontNameAndSize(@"Baskerville-Bold", 15);
+    BONChain *italicChain = BONChain.new.fontNameAndSize(@"Baskerville-Italic", 15);
+
+    BONChain *baseChain = BONChain.new.fontNameAndSize(@"Baskerville", 17)
+                              .tagStyles(@[ BONTagMake(@"bold", boldChain), BONTagMake(@"italic", italicChain) ])
+                              .string(@"<bold>This text is wrapped in a \\<bold> tag.</bold>\n<italic>This text is wrapped in an \\<italic> tag.</italic>");
```

Comment:
#req one of the main points of this feature is to be able to style substrings, and from viewing the example app, it's not obvious that this is the case, because each style is on its own line. It should be something like:

`"<bold>This text</bold> contains a \\<bold> tag.\n<italic>This text</italic> contains an \\<italic> tag."`